### PR TITLE
ModeUtil add checkConditionAssigned()

### PR DESCRIPTION
**Pull request by Billy4195**

### CODE REVIEW

File: modeUtil.py [hunk 1]
```diff
@@ -5,3 +5,10 @@ def checkOrderById(arrangeMode):
             return True
         else:
             return False
+
+    @staticmethod
+    def checkConditionAssigned(arrangeMode):
+        if arrangeMode in [3,4,5,7]:
```
Comment (stvreumi) [inline modeUtil.py]: Why there is a 7 in the list?

Comment (Billy4195) [inline modeUtil.py]: Because the following case mode 7 will also use condition. 
So I added 7 for the future modification.

Comment (stvreumi) [inline modeUtil.py]: But...if `arrangeMode` is `7`, then this function would return 'Ture`. Doesn't it matter?

Comment (Billy4195) [inline modeUtil.py]: Yes, because the mode 7 has been filtered at `if arrange_mode in range(6):`

**Pull request merged**